jobs/JOBSPROC_RAP_DUMP

```diff
@@ -143,7 +145,7 @@ export SENDDBN=${SENDDBN:-YES}
 # Paths for obsproc_network items
 # -------------------------------
 export HOMEobsproc=${HOMEobsproc:\
--${NWROOT:?}/obsproc.${obsproc_ver:?\
+-${PACKAGEROOT:?}/obsproc.${obsproc_ver:?\
```

Comment:
@nicholasesposito @ShelleyMelchior-NOAA , I replaced "obsproc" in all paths with ${obsNET}. Was that a good move or not?
Nick, in line 148, you need "/" instead of "." between obsproc and obsproc_ver, I think.

Reply:
@nicholasesposito , @ilianagenkova 's comment is correct. I suggest this:
export HOMEobsproc=${HOMEobsproc:\
-${PACKAGEROOT:?}/${obsNET}/${obsproc_ver:?\